### PR TITLE
Home page completed 

### PR DESCRIPTION
**Pull request by Adyasha8105**
Fix #554

### CODE REVIEW

**Comment by smaranjitghose** [comment]
wasn't @shreyakapoor08 working on this?

**Comment by smaranjitghose** [comment]
Please try to attach a snapshot the final page whenever you give PRs for this

**Comment by shreyakapoor08** [comment]
> wasn't @shreyakapoor08 working on this?

There were 6 sections, we had divided 3 each. With this PR the documentation of the home page is complete :)

**Comment by smaranjitghose** [comment]
> > wasn't @shreyakapoor08 working on this?
> 
> There were 6 sections, we had divided 3 each. With this PR the documentation of the home page is complete :)

Cool!

File: docs/_build/html/_sources/home.rst.txt [hunk 1]
```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
...
+.. image:: ./images/homepage/trends.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+To make changes in the code follow the below steps :
+
+- To make changes in the heading and the content - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`WE CREATE TRENDS` or :python:`join-us`, you will see <h2> tag or <h3> tag where changes can be done.
+
+* To make changes in the card images - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`ih-item circle effect`, you will see <img> tag where you can change the path of the image according to your image. And accordingly you can change the other images also.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:117]: Specify Sizes of images

File: docs/_build/html/_sources/home.rst.txt [hunk 2]
```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
...
+To make changes in the right part follow the below steps :
+
+- To make changes in the number count - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`ci-number`, and make changes inside the div container and similarly for other number count also.
+
+* To make changes in the heading and content - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`ci-text`, for heading <h4> tag and for content <p> tag and then do the desired changes accordingly.
+
+- To make changes in the box shadow of the container - Go to the file `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__ and search :python:`.myDiv` where you can change the box-shadow by giving appropriate size and color to it.
+
+To make any changes in the animation effect of the whole section - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`sparkley` or :python:`sparkle-canvas`, do your desired changes whether you want or not or any style.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:102]: Rewrite as to modify the animation

File: docs/_build/html/_sources/home.rst.txt [hunk 3]
```diff
@@ -26,7 +26,7 @@ To make any changes in the code follow the below steps.
 
 2. **About**: About section describes about the Girlscript Foundation and their goals.
 
-.. image:: ./images/homepage/aboutus.JPG
+.. image:: ./images/homepage/aboutus.png
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:29]: Always specify the default size of image used

**Comment by shreyakapoor08** [comment]
Noted :)

File: docs/_build/html/_sources/home.rst.txt [hunk 4]
```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
   :align: center
   :alt: Alternative text
 
-- To add some different image go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url. 
+- To add some different image - Go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:55]: Rewrite: For changing the image

Remember the writing style has to be utmost professional 😎

File: docs/_build/html/_sources/home.rst.txt (hunk 3 above)
Comment (Adyasha8105) [reply docs/_build/html/_sources/home.rst.txt:29]: okay sir

File: docs/_build/html/_sources/home.rst.txt [hunk 5]
```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
...
 
-- To add some different image go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url. 
+- To add some different image - Go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url. 
 
 .. note::
    There are three Parallax in the home page.
 
 
-4. **Our Initiatives**:
+4. **Our Initiatives**:  This section describes about the different initiatives taken by girlscript.
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:61]: **taken by your GirlScript Chapter
Keep in mind this documentation is supposed to be a guide addressing various Chapter members in second person

Comment (shreyakapoor08) [reply docs/_build/html/_sources/home.rst.txt:61]: Ohhhhh

Comment (Adyasha8105) [reply docs/_build/html/_sources/home.rst.txt:61]: okay sir

File: docs/_build/html/_sources/home.rst.txt [hunk 6]
```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
...
+
+
+6. **Various Trends**: This section covers various trends upon which contributions have been made.
+
+.. image:: ./images/homepage/trends.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+To make changes in the code follow the below steps :
```
Comment (smaranjitghose) [inline docs/_build/html/_sources/home.rst.txt:113]: Rewrite as:

If you want to make changes in this section, carry out the following changes in code: 

(Repeat for other such lines throughout this page)

Comment (Adyasha8105) [reply docs/_build/html/_sources/home.rst.txt:113]: okay sir

**Comment by Adyasha8105** [comment]
@smaranjitghose 
image

image

image

**Comment by smaranjitghose** [comment]
Update this if the carosel is removed

**Pull request merged**